lib/util/DocumentExtractor.js

```diff
@@ -43,8 +43,18 @@ const extractors = ([
       extractFromRequest: request => {
         const documents = [];
 
-        for (let it = 0; it < request.input.body.ids.length; it++) {
-          documents.push({ _id: request.input.body.ids[it] });
+        if (request.input.body && Object.keys(request.input.body).length) {
```

Comment:
There are more possible execution paths than before in this function  but no unit tests to cover them. Is this intentional?